### PR TITLE
Systemd service and timer to update podcasts

### PR DESCRIPTION
**Pull request by stebe**
I was following issue #2833 and noticed that there are no examples of systemd service and timers.  Here they are.

### CODE REVIEW

**Comment by usox** [comment]
Thanks @stebe. This a nice workaround to avoid problems like #2833

File: docs/examples/ampache_podcasts.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Documentation=https://github.com/ampache/ampache/wiki
+
+[Service] 
+PrivateTmp=true 
+KillMode=mixed
+Type=oneshot
+User=www-data
+Group=www-data
+ExecStart= php bin/cli run:updateCatalog podcasts -a -g -i
+WorkingDirectory=/var/www/ampache
```
Comment (usox) [inline docs/examples/ampache_podcasts.service:13]: Maybe this should match the WorkingDir in ampache_cron.service

Comment (stebe) [reply docs/examples/ampache_podcasts.service:13]: You are correct. I took it from my own setup

**Review by usox — APPROVED** [review]

**Comment by lachlan-00** [comment]
love docs and examples!

**Pull request merged**